src/Control/Effect/Internal.hs

```diff
@@ -138,6 +138,7 @@ type Effs es m = (EffMembers es (Derivs m), Carrier m)
 -- checkString str = send (CheckString str)
 -- @
 --
+-- In most cases you probably want to use 'makeEff' instead.
```

Comment:
Don't discourage using `send`. Not liking Template Haskell is a perfectly fine excuse not to use `makeEff`. `makeEff` should instead be presented as an alternative:
```
-- Alternatively, such definitions can be automatically created for your effect
-- by using 'makeEff'.
```